akka-actor-typed/src/main/scala/akka/actor/typed/Scheduler.scala

```diff
@@ -64,15 +61,18 @@ trait Scheduler {
    * @throws IllegalArgumentException if the given delays exceed the maximum
    * reach (calculated as: `delay / tickNanos > Int.MaxValue`).
    *
-   * Note: For user scheduling needs `Behaviors.withTimers` should be preferred.
+   * Note: For scheduling within actors `Behaviors.withTimers` should be preferred.
    *
-   * Scala API
    */
-  def scheduleAtFixedRate(initialDelay: FiniteDuration, interval: FiniteDuration, runnable: Runnable)(
+  def scheduleWithFixedDelay(
+      initialDelay: FiniteDuration,
+      delay: FiniteDuration,
+      runnable: Runnable)( // FIXME do we want the Runnable as separate parameter list?
```

Comment:
I'd say extra parameter list like in the existing APIs?

Reply:
agreed, changed in 3f3f0f7